src/Storage/Database/Schema/Table/Users.php

```diff
@@ -21,7 +21,7 @@ protected function addColumns()
         $this->table->addColumn('lastseen',       'datetime',   ['notnull' => false, 'default' => null]);
         $this->table->addColumn('lastip',         'string',     ['length' => 32, 'default' => '']);
         $this->table->addColumn('displayname',    'string',     ['length' => 32]);
-        $this->table->addColumn('stack',          'json_array', ['length' => 1024]);
+        $this->table->addColumn('stack',          'json_array', ['length' => 1024, 'notnull' => false]);
```

Comment:
Yes… By ignoring me, partially… It was the `default` array key that broke stuff.

The bigger question is, however, why is this needed? Is this just something that popped up while stack was broken in master (I only fixed it in the session merge)?